### PR TITLE
More misc cleanup to reflect kubefed rename

### PR DESCRIPTION
**Pull request by marun**
Partial fix for #936

### CODE REVIEW

File: cmd/controller-manager/app/controller-manager.go [hunk 1]
```diff
@@ -244,7 +244,7 @@ func setDefaultKubeFedConfig(fedConfig *corev1b1.KubeFedConfig) {
 		// TODO(sohankunkerkar) Remove when no longer necessary.
 		// This Environment variable is a temporary addition to support Red Hat's downstream testing efforts.
 		// Its continued existence should not be relied upon.
-		const defaultScopeEnv = "DEFAULT_FEDERATION_SCOPE"
+		const defaultScopeEnv = "DEFAULT_KUBEFED_SCOPE"
```
Comment (marun) [inline cmd/controller-manager/app/controller-manager.go]: cc: @sohankunkerkar @pmorie

File: cmd/kubefedctl/main.go [hunk 2]
```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// kubefedctl is a tool for managing clusters in a federation.
+// kubefedctl is a tool for managing a KubeFed control plane.
```
Comment (xunpan) [inline cmd/kubefedctl/main.go]: I'm not sure that kubefedctl manages the control plane? It is a tool for federation functions support.

Comment (marun) [reply cmd/kubefedctl/main.go]: I think it's a reasonable generalization to say that kubefedctl manages a control plane given that it supports cluster registration, type enablement, and federation of resources.  In any case, what I have proposed is more accurate than the previous comment.

**Comment by irfanurrehman** [comment]
This LGTM for me. Thanks @marun

**Pull request merged**